.github/workflows/mdbook_cd.yml

```diff
@@ -0,0 +1,29 @@
+name: Spec MDBook CD
+
+on:
+  push:
+    tags: 
+      - '*'
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v3
+
+      - name: Download mdbook build artifact 📦
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: mdbook_ci.yml
+          workflow_conclusion: success
+          branch: main
+          check_artifacts:  true
+
```

Comment:
@wilwade  so here it will download artifacts on main branch so we get the latest commit on main

Reply:
This will also ensure tags creation always picks the main branch's latest artifacts @wilwade

Reply:
@saraswatpuneet Does this mean we still release the latest version even if we tag a previous commit?

Imagining this line of commits from new to old:
E
C
B
A

If I tag C, does the build from E or C get used?